docs/buildah-from.md

```diff
@@ -15,6 +15,10 @@ The container ID of the container that was created.  On error, -1 is returned an
 
 ## OPTIONS
 
+**--creds** *creds*
```

Comment:
--source-creds or --dest-creds - creds doesn't work as I stated here https://github.com/projectatomic/buildah/pull/204/files#r130359101

the FROM instruction for instance uses `--source-creds` while pushing the final artifact to a registry should just use `--dest-creds` and not send source creds to the dest registry.